.ci/ci-deploy.sh

```diff
@@ -6,18 +6,13 @@
 set -e
 
 sudo apt-get update
-sudo apt-get install -y xmlstarlet
-
 
 ### MAVEN CENTRAL
 if [ "$TRAVIS_PULL_REQUEST" = "false" ] && [ "$TRAVIS_BRANCH" = "master" ]; then
     cp .travis/travis-settings.xml $HOME/.m2/settings.xml
     cd src 
     
     # pushing snapshot to https://oss.sonatype.org/content/repositories/snapshots/fr/inria/repairnator/
-    VERSION=`xmlstarlet sel -t -v '//_:project/_:properties/_:revision' pom.xml`
-    sed -i -e 's/\${revision}/'$VERSION'/' pom.xml */pom.xml
```

Comment:
We will probably still need this, so that we deploy with an actual version and not `${revision}`